### ISSUE TITLE
Share the project with community

### ISSUE THREAD

**Issue by vvscode** [Aconex/drakov#189]
@subinvarghesein  @krizalys  @boguan @Tavio @yakovkhalinsky 

Hi, folks

Thanks for the great tool. But as far as I see - you have no time to support it (review PR, update dependencies and so on)

Would you mind to share the project with the community?

**Comment by kamihouse** [comment]
@marcelogo @yakovkhalinsky @galkinrost @MartinNuc ?

**Comment by yakovkhalinsky** [comment]
This project was created while I was employed at Aconex, which last year was acquired by Oracle.

I am not really qualified what this means in regards to the fate of this source code.

Someone who is currently employed at Oracle Aconex would be better positioned to answer this question.

**Comment by vvscode** [comment]
@yakovkhalinsky  can you mention someone from that team?

**Comment by vvscode** [comment]
@aj-jaswanth @liquidboy @marcelogo @Matty666 @nmarsden @tallpsmith @TroyWilsonAus @ykulbak

**Comment by Art4** [comment]
Are there any updates about this topic?

**Comment by tallpsmith** [comment]
I'm still here at Oracle and will be able to provide an official update on this repo soon.   My _intent_ is to get official Oracle blessing to keep this repo open-sourced.  It might require relocation to a new Github location and some tidy up, but I'm certainly trying to keep this one alive (we use this internally still ourselves).

I'll be contacting the original crew here privately (who are no longer with us at Aconex/Oracle) to start socialising the changes.

cheers,

Paul Smith

**Comment by yakovkhalinsky** [comment]
@tallpsmith I would certainly give my 👍 to having this stay open sourced.

Perhaps we need to clarify the license on this repo or the new repo as well.

**Comment by tallpsmith** [comment]
@yakovkhalinsky  thanks - I'll be trying to setup a quick zoom chat with yourself and @marcelogo and anyone else that you might think important in the discussion, and I'll share you my thoughts of what I think might happen, plus I have a number of questions for you ... :-D

**Comment by vvscode** [comment]
Please, keep us updated ))

I created the issue 1.5 year ago, but my request is still valid for a number of developers =)

**Comment by tallpsmith** [comment]
I and a few others are progressively getting this repo back into healthy state.  Thanks to Yakov I now have access to the NPM registry permissions needed to be able to publish to the main NPM registry.  TravisCI is now back in the green, after dropping support for older Node versions.

I am wondering if there is a way we, as a group, can verify when the state of the repo is ready for an official 2.0.0 release to go to the NPM Registry?  If people were able to grab this repo, install it locally and use 'in anger' to confirm nothing has broken, then I'd be happy to publish a new official release as we stand.

**Comment by tallpsmith** [comment]
FYI I published a `2.0.1.beta.0` this morning, using `release-it`.  Oddly I had to go a `.1` here because of the way `release-it` parses the existing package.json's `2.0.0` version.

```❯ npm info drakov@beta

drakov@2.0.1-beta.0 | MIT | deps: 13 | versions: 42
Mock server that implements the API Blueprint specification
https://github.com/aconex/drakov

bin: drakov

dist
.tarball: https://registry.npmjs.org/drakov/-/drakov-2.0.1-beta.0.tgz
.shasum: ec94d504fed45d7fe1f3cb1fb5f6741a2d15b4ad
.integrity: sha512-WFh0KZDgU8LnxOuF9yxZsTHhDlrG/JI+U18b2nyaGySV/Hd6+I1WdlAulfnvJf6d5ZqxeaY13zvIId9Tqg3GUQ==
.unpackedSize: 275.6 kB

dependencies:
async: ^2.1.4          colors: ^1.1.0         express: ^4.16.4       http-shutdown: ^1.2.0  path-to-regexp: ^1.7.0 qs: ^6.3.0             yargs: ^16.1.0
chokidar: ^3.4.3       drafter: ^1.2.0        glob: ^7.1.1           lodash: ^4.17.4        pug: ^2.0.4            tv4: ^1.1.9

maintainers:
- mobz <ben@mobz.org>
- russianator <yakov@therocketsurgeon.com>
- marcelogo <marcelo.mgo@gmail.com>
- subinvarghesein <test@example.com>
- tallpsmith <tallpsmith@gmail.com>

dist-tags:
beta: 2.0.1-beta.0  latest: 1.0.4```

NPM still shows `1.0.4` as current, but I believe you can reference the beta with something like `npm install — save drakov@beta`

If there's anyone that wishes to try that Beta via npm and give me the thumbs up, I'll publish an official `2.0.1`

**Comment by tallpsmith** [comment]
`2.0.1` official release is now public and pushed to NPM.  I'll continue to be the project sponsor for the time being, though there will be some others brought into to picture as well.

@vvscode  I'm going to close this issue as more of a tidy up, but if there's any further issues reach out to me. thanks.
